src/app/api/tools/cowswap/route.ts

```diff
@@ -18,24 +16,14 @@ export async function POST(req: NextRequest): Promise<NextResponse> {
   return handleRequest(req, logic, (result) => NextResponse.json(result));
 }
 
-async function logic(
-  req: NextRequest,
-): Promise<OrderResponse & { data: SwapFTData }> {
+async function logic(req: NextRequest): Promise<OrderResponse> {
   const parsedRequest = await parseQuoteRequest(
     req,
     await getTokenMap(),
     getZerionKey(),
   );
   console.log("POST Request for quote:", parsedRequest);
-  const orderResponse = await orderRequestFlow(parsedRequest);
-  const result = {
-    ...orderResponse,
-    data: parseWidgetData({
```

Comment:
it would be good if we changed only the name from data to ui, not the entire payload response structure. I will have to change the entire payload on frontend and also on runtime because of that.

Reply:
And it will be simplified.